projects/demo/src/modules/experimental/icon/icon.template.html

```diff
@@ -0,0 +1,62 @@
+<tui-doc-page
+    header="Icon"
+    package="EXPERIMENTAL"
+    type="components"
+>
+    <ng-template pageTab>
+        <tui-notification status="warning">
+            This code is
+            <strong>experimental</strong>
+            and is a subject to change. Expect final solution to be shipped in the next major version
+        </tui-notification>
+
+        <p>A component to use icons and color them with CSS.</p>
+
+        <tui-doc-example
+            id="basic"
+            heading="Basic"
+            [content]="example1"
+        >
+            <tui-icon-example-1></tui-icon-example-1>
+        </tui-doc-example>
+
+        <tui-doc-example
+            id="external"
+            heading="External icons"
+            [content]="example2"
+        >
+            <tui-icon-example-2></tui-icon-example-2>
+        </tui-doc-example>
+
+        <tui-doc-example
+            id="duo"
+            heading="Two colors"
+            [content]="example3"
+        >
+            <tui-notification class="tui-space_bottom-4">This example only works in modern browsers</tui-notification>
```

Comment:
Should we add link to `Can i use`?

```suggestion
            <tui-notification class="tui-space_bottom-4">This example only works in <a tuiLink href="https://caniuse.com/?search=mask-mode">modern browsers</a></tui-notification>
```

I think this example can cause many questions from our users (chrome is modern browser but it does not support this feature)?

Reply:
Should we hide this example if it does not support this feature ?

Something like the following:
```less
@supports not (mask-mode: luminance) {
    .all-example {
        display: none;
    }
}
```

Reply:
Good idea. I believe Chrome 120 actually should support it, even though it is not mentioned on caniuse. It comes out in a month, we'll see :)

Reply:
Yes, tested in Canary, it will work starting next major version.